### PR TITLE
chore: pull request template

### PR DESCRIPTION
**Pull request by kevinslin**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Pull Request Checklist
+
+You can go to [dendron pull requests](https://wiki.dendron.so/notes/adc39825-77a6-46cf-9c49-2642fcb4248e.html) to see full details for items in this checklist.
+
+## General
+
+### Quality Assurance
+- [ ] add a [test](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#writing-tests) for the new feature
+- [ ] make sure all the existing [tests](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#running-all-tests) pass
+- [ ] do a spot check by running your feature with our [test Workspace](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#test-workspace)
```
Comment (hikchoi) [inline pull_request_template.md:11]: nitpick: 
```suggestion
- [ ] do a spot check by running your feature with our [test workspace](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#test-workspace)
```

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] after you submit your pull request, check the output of our [integration test](https://github.com/dendronhq/dendron/actions) and make sure all tests pass
+  - NOTE: if you running mac/linux, check the windows output and vice versa if you are developing on windows
+
+#### Special Cases
+- [ ] if your tests changes an existing snaphot, make sure that snapshots are [updated](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#test-workspace)
```
Comment (hikchoi) [inline pull_request_template.md:16]: Wrong link
```suggestion
- [ ] if your tests changes an existing snaphot, make sure that snapshots are [updated](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#updating-test-snapshots)
```

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] if you are adding a new language feature (graphically visible in vscode/preview/publishing), make sure that it is included in [test-workspace](https://github.com/dendronhq/dendron/blob/feat/hashtag-tags/test-workspace/dendron.yml#L1:L1). We use this to manually inspect new changes and for auto regression testiing
```
Comment (hikchoi) [inline pull_request_template.md:17]: the link 404s. maybe you meant this?: 
```suggestion
- [ ] if you are adding a new language feature (graphically visible in vscode/preview/publishing), make sure that it is included in [test-workspace](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#test-workspace). We use this to manually inspect new changes and for auto regression testiing 
```

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+## Special Cases
+
+### First Time PR
+- [ ] sign the [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) which will be prompted by our github bot after you submit the PR
+- [ ] add your [discord](https://discord.gg/AE3NRw9) alias in the review so that we can give you the [horticulturalist](https://wiki.dendron.so/notes/7c00d606-7b75-4d28-b563-d75f33f8e0d7.html#horticulturalist) badge in our community
+
+
+
+### Analytics
+- [ ] if you are adding analytics related changes, make sure the [Telemetry](dendron.ref.telemetry) docs are updated
```
Comment (hikchoi) [inline pull_request_template.md:31]:
```suggestion
- [ ] if you are adding analytics related changes, make sure the [Telemetry](https://wiki.dendron.so/notes/84df871b-9442-42fd-b4c3-0024e35b5f3c.html) docs are updated
```

**Review by hikchoi — CHANGES_REQUESTED** [review]
Fixed some links but other than that looks good.

**Review by hikchoi — APPROVED** [review]

**Comment by kevinslin** [comment]
> Can we add a line in there for release note description?
> 
> Something along the lines of
> 
> > If this is a user-facing feature or may the overall Dendron experience, please summarize the feature or impact in 1-2 lines for the changelog release notes. If this builds upon a previous PR covering the same area, please link to that PR instead.

made some updates

**Review by jonathanyeung — APPROVED** [review]

**Pull request merged**